.gitignore

```diff
@@ -25,3 +25,5 @@ dist/
 public/
 
 /coverage
+
+.env
```

Comment:
do we not need the env file youre adding?

Reply:
It is recommended not to include .evn files into VCS as it might contain sensitive information.

Reply:
we store all of the sensitive info in secrets